app/controllers/admin/organizations_controller.rb

```diff
@@ -4,8 +4,51 @@ def index
       @organizations = Organization.paginate(page: params[:page])
     end
 
+    def new
+      @organization = Organization.new
+      @organization.permissions = OrganizationPermission.new
+    end
+
+    def create
+      @organization = Organization.new(organization_params)
+      @organization.permissions = permissions(OrganizationPermission.new)
+
+      if @organization.save
+        redirect_to admin_organization_path(@organization.id)
+      else
+        render :new
+      end
+    end
+
     def show
-      @organization = Organization.find_by(id: params[:id])
+      @organization = Organization.find(params[:id])
+    end
+
+    def edit
+      @organization = Organization.find(params[:id])
+    end
+
+    def update
+      @organization = Organization.find(params[:id])
+      @organization.permissions = permissions(@organization.permissions)
+
+      if @organization.update(organization_params)
+        redirect_to admin_organization_path(@organization.id)
+      else
+        render :edit
+      end
+    end
+
+    def permissions(organization_permissions)
+      organization_permissions.uphold_wallet = params[:uphold] == "1"
```

Comment:
I believe the default value is `"1"` and you set the value to be `1` if checked. I'm under the impression that `params[:uphold].present?` is more conventional.

Reply:
ah! I forgot that when the input is unchecked it won't send that params
https://github.com/brave-intl/publishers/pull/1417/commits/c18483e3b2a1235d313ab4596da663d713fe7290